src/Zippy.php

```diff
@@ -80,9 +80,11 @@ public function create($path, $files = null, $recursive = true, $type = null)
      *
      * @throws RuntimeException In case of failure
      */
-    public function open($path)
+    public function open($path, $type = null)
     {
-        $type = $this->guessAdapterExtension($path);
+        if($type == null){
```

Comment:
Missing spaces before and after the parentheses. 

See https://github.com/GiantCowFilms/Zippy/blob/e4e42f4f7fd80f44eb256e73e7a78dc714237685/src/Zippy.php#L61 for an example